integrity-backend/input-browsertrix-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-recorder.json

```diff
@@ -71,7 +71,7 @@
           "type": "git",
           "values":
           {
-            "repository": "git@github.com:webrecorder/browsertrix-cloud.git",
+            "repository": "https://github.com/webrecorder/browsertrix-cloud.git",
             "branch": "main",
             "commit": "29b586b03f181d431b2f9c981d92d50725fade07",
             "clean": true
```

Comment:
I have not updated the code to exclude tokes and such, this will be updated in one place.